fedmsg_meta_fedora_infrastructure/conglomerators/ansible/playbooks.py

```diff
@@ -0,0 +1,55 @@
+import fedmsg.meta.base
+from fedmsg_meta_fedora_infrastructure.fasshim import avatar_url
+
+
+class ByUser(fedmsg.meta.base.BaseConglomerator):
+    def can_handle(self, msg, **config):
+        return 'ansible.playbook' in msg['topic']
+
+    def possessive(self, name):
+        """ Given a name, return the possessive form. """
+        return self._("{name}'s").format(name=name)
+
+    def merge(self, constituents, **config):
+        ms = constituents  # shorthand
+
+        agent = ms[0]['msg']['userid']
+        playbooks = set([m['msg']['playbook'].split('/')[-1] for m in ms])
+        playbooks_text = self.list_to_series(playbooks)
+
+        N = len([m for m in ms if m['topic'].endswith('start')])
+        if not N:
+            N = len(ms)
+
+        if len(playbooks) > 1:
+            N = 's %i' % N
+        else:
+            N = ' %i' % N
+
+        subtitle = '{agent} ran the {playbooks} playbook{N} times'
```

Comment:
Should the ``times`` be included in ``N``? Otherwise we may end up with: ``ran the bodhi.yml playbook 1 times``.